### DISCUSSION TITLE
Osu! won't start anymore.

### DISCUSSION THREAD

**Discussion by unisenpai** [ppy/osu#14080] [Q&A]
Hello, 

I am writing this discussion because it has been over a month and I haven't been able to open Osu! no matter what I try to do to fix it. 
When clicking on the app, my cursor loads, and in my task manager Osu! pops up, bu then it quickly disappears and the game never starts. 

I have tried going to the properties checking the compatibilities, changing them and starting over; nothing.

I have also restarted the computer a few times, checked the updates for Windows and my drivers, all is up to date. 
When looking it up online I find nothing that fits my issues and after posting on the Osu! forums and trying what I was told and what I read, nothing worked. Here is what I've collected so far.

Going into Event Viewer does this:
![image](https://user-images.githubusercontent.com/88227216/127733181-f643ea24-5cbf-4bbe-8ea1-d70aabe26a54.png)

then this 
![image](https://user-images.githubusercontent.com/88227216/127733191-2e532c2b-3040-4003-87f6-0d75542034c3.png)

and this
![image](https://user-images.githubusercontent.com/88227216/127733193-ee9593c9-bc49-42d6-a0c5-e8b99c1bfc20.png)

I was answered and asked if my .NET runtime was up to date, it was, but to make sure I went and downloaded the newest version, restarted the PC and still the same issue. 
This issue is with every single version I tried to install from new to old.

Going into %appdata% and checking if there was anything weird going on, there wasn't. 

I am posting this hoping you could help me find a solution and that what I've provided is good information.

**Comment by peppy** [comment]
You haven't provided any details on what version of windows you are running, or what version of osu! is failing.

**Reply by unisenpai** [reply]
Sorry! 

For Windows it's: 
Édition	Windows 10 Famille
Version	21H1
Installé le	‎05/‎05/‎2021
Build du système d’exploitation	19043.1110
Numéro de série	MP1VVGLX
Expérience	Windows Feature Experience Pack 120.2212.3530.0

And for osu! it's like I said, every version I try doesn't start, but I'd say the one I've tried the most is the newest release (2021.724.1)
and also (2021.720.0) and (2021.707.0).

**Comment by bdach** [comment]
Do you have an nvidia graphics card? If so please try disabling shadowplay.

**Reply by unisenpai** [reply]
I don't, this is what I have:
![image](https://user-images.githubusercontent.com/88227216/127734227-8fcf15a0-3c9d-42f7-9314-381a01cf0c96.png)

**Reply by bdach** [reply]
In that case please try updating your graphics card drivers and seeing if it helps. Attaching the event viewer errors in full as an exported report (rather than just posting single screenshots) could also be helpful for us to diagnose.

**Reply by unisenpai** [reply]
So I went into them and clicked on updating to the newest drivers but it said it already the best ones installed, so then it gives me the prompt to look for updates with Windows or something along those lines, so I clicked and it gave me a new update to do for Windows which I did, this is what it said in case it helps: 
![image](https://user-images.githubusercontent.com/88227216/127734508-64b19e9d-0c1d-4a32-bae3-6fd8d7ce2a6d.png)

and for the Event viewer I don't understand what you mean by an exported report could you please be a bit more specific, thank you.

**Reply by bdach** [reply]
.NET Framework should have nothing to do with it.

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by unisenpai** [reply]
The reply says I can't insert an .evtx file into it, should I make a zip instead?

**Reply by frenzibyte** [reply]
Yes.

**Reply by unisenpai** [reply]
[osu errors.zip](https://github.com/ppy/osu/files/6910952/osu.errors.zip)

**Reply by bdach** [reply]
Unfortunately there is nothing to go on in the attached file. Short of debugging it yourself or getting a crash dump we will not be able to proceed with diagnosing further.

Note: do NOT upload or link a crash dump here as it may leak sensitive information.

**Reply by unisenpai** [reply]
What do I do then?

**Comment by smoogipoo** [comment]
Are you using 32-bit Windows?

**Reply by unisenpai** [reply]
Uh no, if I'm correct it's 64-bit
